content/en/apps/tutorials/multi-facility-users.md

```diff
@@ -0,0 +1,202 @@
+---
+title: "Setting up Multi-facility users"
+linkTitle: Multi Facility Users
+weight: 6
+description: >
+  Creating and assigning users to multiple places in the CHT UI
+relatedContent: >
+  apps/concepts/users
+  apps/reference/app-settings
+  apps/tutorials/contact-and-users-1
+---
+
+{{% pageinfo %}}
+This tutorial will take you through how to create and assign users to multiple places in the CHT UI, including;
+
+- Create contacts and their associated users
+- Create places and assign contacts to those places
+- Assigning users to multiple places
+
+App settings allow you to both persist information that is critical to the application outside the code, and to create profiles that store the preferences for project deployments.
+{{% /pageinfo %}}
+
+## Brief Overview of Key Concepts
+
+*[Permissions]({{< relref "apps/concepts/users#permissions" >}})* are settings that control access to specific app features and functionality.
+
+*[Roles]({{< relref "apps/concepts/users#roles" >}})* define permissions for users to access a group of app features and functionality.
+
+*[Contacts]({{< relref "apps/features/contacts" >}})*  are people or places that are created in the CHT application.
+
+*[People]({{< relref "apps/features/contacts" >}})* are both patients in the system and users of the system, such as CHWs or Nurses.
+
+*[Places]({{< relref "apps/features/contacts" >}})* represent either an actual physical location such as a health facility, clinic, or a grouping such as a household or CHW Area.
+
+## Required Resources
+
+- You should be familiar with contact and user management. View detailed tutorial on [How to create contacts and their associated users]({{% ref "apps/tutorials/contact-and-users-1" %}}) 
+- You should be familiar with managing CHT application settings. Read [How to manage CHT application settings]({{% ref "apps/tutorials/application-settings" %}}) 
+
+
+## Implementation Steps
+
+In this tutorial, you will work with the default contact forms and the default hierarchy.
+
+The default `app_settings.json` in the CHT found in the `/config/default` folder has an existing `chw_supervisor` role and the `"can_have_multiple_places": []` permission. 
+
+While logged in as an admin user, you will create two Health Facilities, CHW Supervisor, and CHW Area. You will then create the user for the CHW Supervisor so that they can log in see their facilities, and the CHW Areas they supervise.
```

Comment:
```suggestion
While logged in as an admin user, you will create two Health Facilities, CHW Supervisor, and CHW Area. You will then create the user for the CHW Supervisor so that they can log in and see the facilities, and the CHW Areas they supervise.
```